lib/LaTeXML/Package.pm

```diff
@@ -2285,7 +2286,7 @@ sub AddToMacro {
   else {
     DefMacroI($cs, undef, Tokens(map { $_->unlist }
           map { (blessed $_ ? $_ : TokenizeInternal($_)) } ($defn->getExpansion, @tokens)),
-      scope => 'global'); }
+      nopackParameters => 1, scope => 'global'); }
```

Comment:
Right, so the perl `AddToMacro` tries to emulate a latex technique which leverages `\the`. It is completely natural that it would need this guard added to it, and we (= I) may want to grep over latex.ltx to find other cases using `\the` that we have reimplemented in perl. They'll need that flag as well.